Umbraco-Cloud/Security/index.md

```diff
@@ -0,0 +1,46 @@
+# Security on Umbraco Cloud
+
+## HTTPS & Certificates
+
+All Umbraco Cloud websites use HTTPS by default. Both the default {projectName}.s1.umbraco.io and custom domains are protected by periodically renewed certificates from Let's Encrypt. This service is offered as part of Umbraco Cloud by [Umbraco LATCH](/Umbraco-Cloud/Set-Up/Umbraco-Latch/index.md)
+
+### Custom Certificates
+
+Custom certificates can be used with all custom domains. Please refer to [Managing Custom Certificates documentation](Umbraco-Cloud/Set-Up/Manage-Hostnames/Security-Certificates/index.md)
+
+### TLS support
+
+As of April 2020, we've deprecated support for TLS1.0 & TLS 1.1. TLS1.2 is the only supported TLS protocol going forward.
```

Comment:
Should this say "minimum supported version" ?

Reply:
Should we explicitly state that we add support for TLS 1.2 for the older .net framework versions where TLS 1.0 was the default and TLS 1.2 wasn't added by default? I believe this is the case for sites on Cloud running Umbraco 7.x and .net framework 4.5.2.

Reply:
@sitereactor we only support TLS1.2, we don't support 1.3, that means 1.2 is the only supported version. We could remove "only".

I've now specifically mentioned changing the default TLS version for version 7 websites.

Reply:
Sure I get that. I just thought that "minimum" would be better, so we don't have something that is potentially wrong for certain parts of the platform when the hosting changes or when we have a better DNS/CDN provider in front. But fine to leave this for now.